src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -1471,7 +1470,6 @@ def test_technical_on(self):
         response = self.client.post(self.technical_edit_url, data)
         addon = self.get_addon()
 
-        assert not addon.view_source
```

Comment:
When this part is deleted, the relevant upper line should also be considered.

Reply:
I guess you imply `addon = self.get_addon()`?

Reply:
I think that 1469-1471 lines are preceded for work 1474 line.
So, I mentioned my opinion.